example-widget-mui/src/IdentityPage/IdentityPage.test.tsx

```diff
@@ -55,8 +55,14 @@ describe('<IdentityPage />', () => {
     await expect(
       screen.findByRole('heading', { name: /Identity/i }),
     ).resolves.toBeInTheDocument();
-    expect(screen.getByText('Error')).toBeInTheDocument();
-    expect(screen.getByText(/error: .+/i)).toBeInTheDocument();
+
+    await waitFor(() => {
```

Comment:
`waitFor` is not needed here. Instead just change

```
expect(screen.getByText('Error')).toBeInTheDocument();
```

to

```
expect(await screen.findByText('Error')).toBeInTheDocument();
```

Can you also check other places where this may apply?

Reply:
Fixed all simular occurrences.